docs/source/notebooks_and_ipython/kedro_and_notebooks.md

```diff
@@ -236,6 +236,37 @@ If a Jupyter kernel with the name `kedro_<package_name>` already exists then it
 
 You can use the `jupyter kernelspec` set of commands to manage your Jupyter kernels. For example, to remove a kernel, run `jupyter kernelspec remove <kernel_name>`.
 
+### Debugging with %debug and %pdb
+
```

Comment:
It's a really long sentence, is there a way to shorten this, perhaps by using bullets?